reference/shipping.v2.yml

```diff
@@ -1400,6 +1434,25 @@ paths:
             "shipping_cost": 8
         }
         ```
+        
+        ### Channels
+
+                Example request body: 
```

Comment:
Looks like this block has an extra level of indentation compared to the `Range Object – Properties` example above?